File: 백준/Bronze/30957. 빅데이터 vs 정보보호 vs 인공지능/빅데이터 vs 정보보호 vs 인공지능.c
#include <stdio.h>
#include <string.h>

int main() {
    int N;
    scanf("%d", &N);
    
    char input[N + 1];
    scanf("%s", input);
    
    int counts[3] = {0};
    for (int i = 0; i < N; i++) {
        if (input[i] == 'B') {
            counts[0]++;
        } else if (input[i] == 'S') {
            counts[1]++;
        } else if (input[i] == 'A') {
            counts[2]++;
        }
    }

    if (counts[0] == counts[1] && counts[1] == counts[2]) {
        printf("SCU\n");
    } else {
        int maxCount = counts[0];
        for (int i = 1; i < 3; i++) {
            if (counts[i] > maxCount) {
                maxCount = counts[i];
            }
        }
        if (counts[0] == maxCount) {
            printf("B");
        }
        if (counts[1] == maxCount) {
            printf("S");
        }
        if (counts[2] == maxCount) {
            printf("A");
        }
    }
    return 0;
}

File: 백준/Bronze/30957. 빅데이터 vs 정보보호 vs 인공지능/빅데이터 vs 정보보호 vs 인공지능.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int N = scanner.nextInt();
        scanner.nextLine();
        
        int B = 0, S = 0, A = 0;
        String input = scanner.nextLine();
        for (char c : input.toCharArray()) {
            if (c == 'B') {
                B++;
            } else if (c == 'S') {
                S++;
            } else if (c == 'A') {
                A++;
            }
        }

        if (B == S && S == A && B == A) {
            System.out.println("SCU");
        } else {
            if (B == Math.max(B, Math.max(S, A))) {
                System.out.print("B");
            }
            if (S == Math.max(B, Math.max(S, A))) {
                System.out.print("S");
            }
            if (A == Math.max(B, Math.max(S, A))) {
                System.out.print("A");
            }
        }
        scanner.close();
    }
}
